src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -656,7 +656,7 @@ public override async Task<IReadOnlyList<BulkStateItem>> GetBulkStateAsync(strin
                 await this.MakeSaveStateCallAsync(storeName, key, value, etag, stateOptions, metadata, cancellationToken);
                 return true;
             }
-            catch (RpcException)
+            catch (RpcException rpc) when (rpc.StatusCode == StatusCode.Aborted && (rpc.Status.Detail.Contains(ETagInvalid) || rpc.Status.Detail.Contains(ETagMismatch)))
```

Comment:
We should not be comparing error messages here - checking the status code is enough.

Remember, these are user-facing descriptive error messages. If we update them to change the phrasing we don't want this code to break. Using the status code is enough 😄

Reply:
That's why I am not comparing the entire error string but I thought the constants should not change. I hope Aborted error code does not get used for something else because it still sounds pretty generic. Thoughts?

Reply:
If a string in code is readable as a sentence (like these are) then it's for humans. Strings for computers look like `ERR_STATE_CONFLICT` or something similar.

The gRPC Aborted status code has a **very** specific meaning, and it's what we should rely on. The name sounds generic but it has a precise meaning that maps to our use case.  https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Reply:
ok...makes sense